src/lib/page-state/page-state.ts

```diff
@@ -17,7 +17,7 @@ declare global {
 /**
  * @tag forge-page-state
  *
- * @summary Page states display full-page messages for empty states, errors, or loading scenarios.
+ * @summary Page states display full-height messages for empty states, errors, or loading scenarios. Can be used within a full page or within a container.
```

Comment:
I'd make the second sentence a complete sentence, just to match the tone of the rest of the docs